CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Serilog Message Toolbox
 
+# -----------Deprecated-----------
```

Comment:
Hier zou ik nog wat bijkomende info bijzetten.

```
## Deprecated

This toolbox is no longer maintained because it seemed to be unused and the `MessageVersion` log field is not required according to the [Digipolis logging guidelines](https://github.com/digipolisantwerpdocumentation/logging-requirements).
```